build/root/.bazelrc

```diff
@@ -29,7 +29,8 @@ test:unit --features=race
 
 test:unit --build_tests_only
 test:unit --test_tag_filters=-e2e,-integration
-test:unit --flaky_test_attempts=3
+test:unit --runs_per_test=20
+test:unit --cache_test_results=no
 
 test:integration --local_test_jobs 4
 test:integration --test_tag_filters=integration
```

Comment:
integration doesn't work anyhow IIRC, but I think we probably wanted flaky_test_attempts on this instead